netbox/dcim/tables/devices.py

```diff
@@ -872,7 +872,8 @@ class ModuleBayTable(DeviceComponentTable):
     )
     module_status = columns.TemplateColumn(
         verbose_name=_('Module Status'),
-        template_code=MODULEBAY_STATUS
+        template_code=MODULEBAY_STATUS,
+        orderable=False
```

Comment:
We should be able to support ordering for this column if we define an accessor for it. `A('installed_module__status')` should work, I think.